docs/templates/generic_child_template.rst.jinja

```diff
@@ -26,7 +26,7 @@ described here.
 {% block download_docs %}
 This block iterates through any downloadable PDFs for the data source that might provide
 extra context.  These might be a diagram of the database or a blank form for the data
-source for a certain time frame. Place documents in ``docs/data_sources/{{ resource.name }}/`` and they will show up here automatically.
+source for a certain time frame. Place documents in ``docs/data_sources/{{ source.name }}/`` and they will show up here automatically.
```

Comment:
censuspep doesn't have any documents, but this failed in the first smoketest build after copying the template and before making any changes so I figured might as well fix it